src/amr/data/particles/refine/split.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef PHARE_SPLIT_H
```

Comment:
do we care that this isn't "PHARE_SPLIT_HPP" ?

Reply:
do we?

Reply:
some tools complain about these things

:man_shrugging:

Reply:
do you have these "some" in mind? I've never seen a tool that cares about the header guard

Reply:
cpplint is such a tool, but we don't use it atm

```
Done processing src/python3/cpp_simulator.h
src/python3/data_wrangler.h:0:  No copyright message found.  You should have a line: "Copyright [year] <Copyright Owner>"  [legal/copyright] [5]
src/python3/data_wrangler.h:1:  #ifndef header guard has wrong style, please use: SRC_PYTHON3_DATA_WRANGLER_H_  [build/header_guard] [5]
```

Reply:
done